documentation/docs/how_tos/outputs_and_transactions/06_check_unlock_conditions.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Check Unlock Conditions
+description: 'How to check unlock conditions of an output.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- check unlock conditions
+- output
+- nodejs
+- python
+- java
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
+import PythonCode from "../../libraries/python/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import PythonOutput from "../../libraries/python/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
+import JavaCode from "../../libraries/java/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import JavaOutput from "../../libraries/java/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
+import RustCode from "../../libraries/rust/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
```

Comment:
Since all the examples have the same output :), we can probably skip this and use a simple code block in this doc. WDYT?
```suggestion
```